Lib/doctest.py

```diff
@@ -1309,7 +1315,7 @@ def _failure_header(self, test, example):
             out.append('File "%s", line %s, in %s' %
                        (test.filename, lineno, test.name))
         else:
-            out.append('Line %s, in %s' % (example.lineno+1, test.name))
+            out.append(f'Line {example.lineno+1}, in {test.name}')
```

Comment:
same here -- I think it's fine to switch to f-strings if we're touching the line anyway, but doing it to unrelated lines as part of this PR just makes the diff confusing, I feel :(